### PR TITLE
Add AR hit test support to status.

### PR DESCRIPTION
**Pull request by jpmedley**
**CC:** @petele
[WF_IGNORE:MAX_LEN,SCRIPT]

### CODE REVIEW

**Comment by jpmedley** [comment]
Just staging this for Thursday.

**Comment by jpmedley** [comment]
@petele This one is ready for you.

File: src/content/en/fundamentals/vr/status/index.md [hunk 1]
```diff
@@ -2,37 +2,39 @@ project_path: /web/fundamentals/_project.yaml
...
+# WebVR Status {: .page-title }
 
 ## WebVR Implementation Status
 
-### WebXR Device API (formerly WebVR 2.0) {:#xrdevice}
+### WebXR Device API {:#xrdevice}
 
-Today the API is available in:
+* Using the WebXR device API requires a
+[compatible device](https://pr-6258-dot-web-central.appspot.com/ar/discover/supported-devices)
```
Comment (petele) [inline src/content/en/fundamentals/vr/status/index.md]: This should not be to the web-central URL

File: src/content/en/fundamentals/vr/status/index.md [hunk 2]
```diff
@@ -2,37 +2,39 @@ project_path: /web/fundamentals/_project.yaml
...
+| VR use cases | Chrome 66 and later | Enable the chrome://flags/#webxr flag. (The URL must be entered manually.) |
+| VR use cases | Chrome 67 origin trial | Enable the chrome://flags/#webxr flag *and* sign up for the origin trial ([explainer](https://github.com/GoogleChrome/OriginTrials/blob/gh-pages/developer-guide.md), [sign-up form](http://bit.ly/OriginTrialSignup)). |
 
-Find more information on browser implementation status on
-[chromestatus.com](https://www.chromestatus.com/features/5680169905815552).
+Learn more about the immersive web at the [Immersive Web Community Group](https://github.com/immersive-web).
 
 ### Version 1.1 {:#version_1_1}
 
+**Note:** This API is deprecated and its origin trial is scheduled to end in
```
Comment (petele) [inline src/content/en/fundamentals/vr/status/index.md]: Do not wrap notes in bold, use standard callout syntax.

**Comment by jpmedley** [comment]
"Do you also want to use this opportunity to move WebVR/AR to a top level node?" 

Not yet.

Also, PTAL.

**Pull request merged**